pkg/basicserver/metrics.go

```diff
@@ -16,22 +16,27 @@ package server
 
 import "github.com/prometheus/client_golang/prometheus"
 
-const (
-	namespace       = "scheduling"
-	serverSubsystem = "server"
-)
-
 var (
-	// Meta & Server info.
-	serverInfo = prometheus.NewGaugeVec(
+	// ServerMaxProcsGauge record the maxprocs.
+	ServerMaxProcsGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: "pd",
+			Subsystem: "service",
+			Name:      "maxprocs",
+			Help:      "The value of GOMAXPROCS.",
+		})
+
+	// ServerInfoGauge indicates the pd server info including version and git hash.
+	ServerInfoGauge = prometheus.NewGaugeVec(
```

Comment:
Another option, we can define the metrics for different services here.

Reply:
One metrics is maybe enough. We can get instance roles from service-role plane and find version info by instance.